### PR TITLE
Rev build stacks and CT-278 updated docs for multiple provisioning profiles 

### PR DESCRIPTION
**Pull request by lm3m**

### CODE REVIEW

File: src/pages/appflow/package/credentials.md [hunk 1]
```diff
@@ -133,3 +134,12 @@ Provisioning profiles give your app access to be installed, or *provisioned*, on
 * Select the certificate you generated.
 * Select any and all development devices.
 * Download the `.mobileprovision` file, which is your provisioning profile file.
+
+#### Multiple Provisioning Profiles
+For apps that have extensions and require multiple provisioning profiles (apps with watch apps for example), these are the requirements to build them in AppFlow:
+
+* Multiple profiles are only supported for [Capacitor](https://capacitor.ionicframework.com) apps
+* That there is one project, one workspace, and one scheme that are all named the same
```
Comment (giocalitri) [inline src/pages/appflow/package/credentials.md:142]: remove the initial `That`?

File: src/pages/appflow/build-stacks.md [hunk 2]
```diff
@@ -18,17 +18,36 @@ Below are the versions used in Appflow for each build type...
...
+| Node.js      | 12.16.1         | 12.17.0         | 12.18.3         | 12.18.3         | 14.15.0         |
+| npm          | 6.13.4          | 6.14.4          | 6.14.6          | 6.14.6          | 6.14.8          |
+| Yarn         | 1.22.0          | 1.22.4          | 1.22.4          | 1.22.5          | 1.22.10         |
+| macOS        | 10.14.5         | 10.14.6         | 10.15.6         | 10.15.6         | 10.15.7         |
+| Carthage     | 0.34.0          | 0.34.0          | 0.35.0          | 0.36.0          | 0.36.0          |
+| CocoaPods    | 1.8.4           | 1.7.5           | 1.9.3           | 1.9.3           | 1.10.0          |
+| Xcode        | 10.2.1          | 11.5            | 11.6            | 12.0            | 12.1            |
+| Debian       | -               | -               | -               | -               | -               |
+| Gradle       | -               | -               | -               | -               | -               |
+| OpenJDK      | -               | -               | -               | -               | -               |
```
Comment (ltm) [inline src/pages/appflow/build-stacks.md:36]:
```suggestion
```

File: src/pages/appflow/build-stacks.md [hunk 3]
```diff
@@ -18,17 +18,36 @@ Below are the versions used in Appflow for each build type...
...
+| -----------  | --------------- | --------------- | --------------- |
+| Cordova CLI  | 9.0.3           | 10.0.0          | 10.0.0          |
+| Ionic CLI    | 6.2.2           | 6.12.1          | 6.12.1          |
+| Node.js      | 12.19.0         | 12.19.0         | 14.15.0         |
+| npm          | 6.14.8          | 6.14.6          | 6.14.8          |
+| Yarn         | 1.22.5          | 1.22.5          | 1.22.5          |
+| macOS        | -               | -               | -               |
+| Carthage     | -               | -               | -               |
+| CocoaPods    | -               | -               | -               |
+| Xcode        | -               | -               | -               |
```
Comment (ltm) [inline src/pages/appflow/build-stacks.md:50]:
```suggestion
```

File: src/pages/appflow/package/credentials.md [hunk 4]
```diff
@@ -133,3 +134,12 @@ Provisioning profiles give your app access to be installed, or *provisioned*, on
...
+* One certificate is used for all provisioning profiles (and that is the certificate for the app in AppFlow)
```
Comment (ltm) [inline src/pages/appflow/package/credentials.md:143]:
```suggestion
* One certificate is used for all provisioning profiles (and that is the certificate for the app in Appflow)
```

File: src/pages/appflow/package/credentials.md [hunk 5]
```diff
@@ -133,3 +134,12 @@ Provisioning profiles give your app access to be installed, or *provisioned*, on
...
+* Each extension is a separate target
+* WidgetKit for app widgets is only supported on xcode 12 or newer
```
Comment (ltm) [inline src/pages/appflow/package/credentials.md:145]:
```suggestion
* WidgetKit for app widgets is only supported on Xcode 12 or newer
```

**Review by ltm — APPROVED** [review]

**Pull request merged**